### PR TITLE
feature_PRESIDECMS-915 ::

### PR DESCRIPTION
**Pull request by cfmitrah**
- Remove unwanted \n \t from json ajaxproxy results.

### CODE REVIEW

File: system/services/helpers/JQueryDatatablesHelpers.cfc [hunk 1]
```diff
@@ -29,6 +29,9 @@ component {
 			ArrayAppend( result.aaData, record );
 		}
 
+		if( isStruct( result.aaData[1] ) AND result.aaData[1]._options NEQ '' )
```
Comment (DominicWatson) [inline system/services/helpers/JQueryDatatablesHelpers.cfc]: I'm not sure about this. Firstly:

* This presumably should be run against each row
* The cleanup needs to be a little smarter than just removing all whitespace?

I imagine the whitespace cleanup would be something like:

```reReplace( result.aaData[ i ]._options, "\s+", " ", "all" )

(i.e. tidyup double spaces, not get rid of all).

Would be good to see some before and after output after making this fix.

**Review by DominicWatson — CHANGES_REQUESTED** [review]
Hi guys, thanks for this - could you take a look at the review feedback?

**Comment by cfmitrah** [comment]
- Functionality optimized. ( now it's  run against each row )
- yes that functionality removing all white space.
- I've attached screen samples for before and after changes.
**Before change :** 
before

**After change :**

after

Please check that and let me know if any update needed ?

**Comment by DominicWatson** [comment]
The issue with removing _all_ whitespace is that text gets broken. There can be editorial text in the return (as in your example) and all the spaces are removed from that.

So, yes - needs to not remove _all_ whitespace, but rather convert 1 or more consecutive whitespace characters into a single whitespace character.

**Comment by DominicWatson** [comment]
Garrrr, this got auto closed because the target branch has disappeared. Really sorry for this. You should be able to re-open from your branch though by targeting the `stable` branch.

**Pull request closed without merging**